src/index.html

```diff
@@ -11,7 +11,244 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post"
+    >
+      <fieldset>
+
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input 
+          type="text"  
+          autocomplete="off"
+          minlength="3"
+          maxlength="15"
+          required
+          >
```

Comment:
```suggestion
          <input 
            type="text"  
            autocomplete="off"
            minlength="3"
            maxlength="15"
            required
          >
```
fix indentation evwrywhere